### PR TITLE
[OBMIN-255] Redesing ProductCard

### PR DESCRIPTION
**Pull request by Kyslytsia**
Redesine ProductCard

### CODE REVIEW

File: src/components/product-card/styles.ts [hunk 1]
```diff
@@ -1,127 +1,306 @@
...
-  -webkit-box-shadow: 0 7px 10px 0 rgba(48, 50, 50, 0.5);
-  -moz-box-shadow: 0 7px 10px 0 rgba(48, 50, 50, 0.5);
-  box-shadow: 0 7px 10px 0 rgba(48, 50, 50, 0.5);
+  width: 164px;
+  height: 260px;
+  border-radius: 10px;
 
   ${({ theme, margin }) => css`
     background-color: ${theme.colors.white};
     ${margin && `margin: ${margin}`};
```
Comment (Wolshebnik) [inline src/components/product-card/styles.ts:28]: move above background

File: src/components/product-card/styles.ts [hunk 2]
```diff
@@ -1,127 +1,306 @@
...
+        top: -2px;
+        left: -2px;
+        right: -2px;
+        bottom: -2px;
+        border-radius: 20px;
+        background: linear-gradient(
+          45deg,
+          transparent 30%,
+          white,
+          white,
```
Comment (Wolshebnik) [inline src/components/product-card/styles.ts:58]: try to change white color to theme.colors.white

File: src/components/product-card/styles.ts [hunk 3]
```diff
@@ -1,127 +1,306 @@
...
+  position: absolute;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: center;
+  background-color: white;
```
Comment (Wolshebnik) [inline src/components/product-card/styles.ts:175]: use color from config

File: src/components/product-card/styles.ts [hunk 4]
```diff
@@ -1,127 +1,306 @@
...
+      line-height: 24px;
+    `}
+  `}
 `;
+
+// export const ButtonBlock = styled.div`
+//   display: flex;
+//   margin: 0 auto;
+//   order: 3;
+// `;
```
Comment (Wolshebnik) [inline src/components/product-card/styles.ts:306]: remove comments

**Review by Voloshkina — APPROVED** [review]

**Review by Mish-hash — APPROVED** [review]

**Review by Wolshebnik — APPROVED** [review]

**Review by Viktoria-Serkiz — APPROVED** [review]

**Review by cloudmenson — APPROVED** [review]

**Review by ASGladchenko — APPROVED** [review]

**Pull request merged**